internal/sapsystem/sapsystem.go

```diff
@@ -77,6 +77,19 @@ type CustomCommand func(name string, arg ...string) *exec.Cmd
 
 var customExecCommand CustomCommand = exec.Command
 
+func (systems SAPSystemsList) GetSIDs() string {
+	var sysNames string
```

Comment:
I would rename the func name to `GetSIDsString` maybe
Why not simply (`Join` takes care to put not putting the separator at the end):
```
        var sids []string
	for _, sid := range systems {
	  sids = append(sids, system.SID)
	}
	return strings.Join(sids, ",")
```